ygen/codegen.go

```diff
@@ -616,7 +616,7 @@ func processModules(yangFiles, includePaths []string, options yang.Options) ([]*
 	var errs util.Errors
 	for _, name := range yangFiles {
 		if err := moduleSet.Read(name); err != nil {
```

Comment:
optional: here you could do errs = util.AppendErr(errs, moduleSet.Read(name))

Reply:
Thanks, done.